packages/vow/vat.js

```diff
@@ -7,7 +7,9 @@
 // @ts-check
 import { isUpgradeDisconnection } from '@agoric/internal/src/upgrade-api.js';
 import { makeHeapZone } from '@agoric/base-zone/heap.js';
-import { makeE, prepareVowTools as rawPrepareVowTools } from './src/index.js';
+
+import { prepareVowToolsForTesting } from './src/tools.js';
```

Comment:
We should not have exports marked "ForTesting" imported in non-test code. 

The was previously distinguished as "raw". I suggest that as the name of the VowTools that is not resumable.
```suggestion
import { prepareRawVowTools } from './src/tools.js';
```
or perhaps "basic" or "simple". Some qualifier so that there is no canonical VowTools and consumer know they have to pick a kind.

Reply:
> perhaps "basic" or "simple"
That's a good suggestion: `prepareBaseVowTools` SGTM.  I'll make that happen.